app/models/ability.rb

```diff
@@ -150,6 +150,11 @@ def initialize(user)
       end
     end
 
+    if feature_enabled? :projects
+      can :read, Project do |project|
+        user && (user.is_pro_admin? || project.member?(user))
+      end
+    end
```

Comment:
I think we risk overloading this ability.rb file with permisions for non-Pro, Pro, Projects and Public tokens. Happy to add this here for now but raise as a concern and a possible refactor in the future.